lightning/src/ln/peers/handshake/acts.rs

```diff
@@ -0,0 +1,38 @@
+/// Wrapper for the first act message
```

Comment:
Frankly, just easier navigation through the components in the file system.

Reply:
Given these are small and either are implementation details or form the public interface of handshake/mod.rs, I'd prefer if they were all in the same file. Otherwise, navigation between files is a bit excessive, IMHO.

Reply:
I would prefer to not stuff the handshake logic with all the accessory structs. I would be open to having one separate file where both the enums/states and the acts would live?

Reply:
For what it's worth, so does the Act enum